### ISSUE TITLE
WCAG 1.1.1: Ensure <img> elements have alternative text or a role of none or presentation (.FooterLegalCMSImg)

### ISSUE THREAD

**Issue by Erikapp** [Erikapp/accessibility-test#28]
#### Issue
`Ensure <img> elements have alternative text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[Aeroméxico](https://www.aeromexico.com/pt-br)
#### Element path
.FooterLegalCMSImg
#### Snippet
`<img class="FooterLegalCMSImg" src="/cms/sites/default/files/2023-11/pci_2.png">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 137.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
